nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/CoreFeature.java

```diff
@@ -282,12 +281,11 @@ protected void cleanupSession(FeaturesRunner runner) {
                 CoreScope.INSTANCE.exit();
             }
         });
-        releaseCoreSession();
         cleaned = true;
     }
 
-    protected void batchRemoveDocuments(IterableQueryResult results) {
-        String rootDocumentId = session.getRootDocument().getId();
+    protected static void batchRemoveDocuments(CoreSession adminSession, IterableQueryResult results) {
```

Comment:
Why static?

Reply:
because as I don't use anything in the feature and to avoid error by confusing adminSession and session, I will remove it